src/library-authoring/create-library/index.scss

```diff
@@ -1,3 +1,8 @@
+@mixin line-height($fontSize: 16) {
+    line-height: $fontSize + px;
+    line-height: ($fontSize/10) + rem;
```

Comment:
Just checking: is this a polyfill?

Reply:
i'm not sure, it was being used in this modal so i moved it from vendor to here

i'll gladly remove it and replace the usages with a simple `line-height: 28px` if you'd prefer